app/forms/instance-create.tsx

```diff
@@ -669,6 +757,106 @@ const AdvancedAccordion = ({
             />
           )}
         </div>
+
+        <div className="flex flex-1 flex-col gap-4">
+          <h2 className="text-sans-md">
+            Floating IPs{' '}
+            <TipIcon>
+              Floating IPs exist independently of instances and can be attached to and
+              detached from them as needed.
+            </TipIcon>
+          </h2>
+          {isFloatingIpAttached && (
+            <MiniTable.Table>
+              <MiniTable.Header>
+                <MiniTable.HeadCell>Name</MiniTable.HeadCell>
+                <MiniTable.HeadCell>IP</MiniTable.HeadCell>
+                {/* For remove button */}
+                <MiniTable.HeadCell className="w-12" />
+              </MiniTable.Header>
+              <MiniTable.Body>
+                {attachedFloatingIpsData.map((item, index) => (
+                  <MiniTable.Row
+                    tabIndex={0}
+                    aria-rowindex={index + 1}
+                    aria-label={`Name: ${item.name}, IP: ${item.ip}`}
+                    key={item.name}
+                  >
+                    <MiniTable.Cell>{item.name}</MiniTable.Cell>
+                    <MiniTable.Cell>{item.ip}</MiniTable.Cell>
+                    <MiniTable.Cell>
+                      <button
+                        onClick={() => {
+                          detachFloatingIp(item.name)
+                        }}
+                      >
```

Comment:
OMG. Found the culprit. The default type of `button` is `submit`, believe it or not. Which is ridiculous. Adding `type="button"` fixes it. I'm going to see if there's a lint rule we can turn on for this, requiring a type on all buttons.

```suggestion
                      <button type="button" onClick={() => detachFloatingIp(item.name)}>

```